workflow/log-monitoring/log-explorer.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Log Explorer
+sidebarTitle: Log Explorer
+---
+
+The Log Explorer is where all of your historical log data is located, including the date and time the log was fired as well as its corresponding resource attributes. This section is useful for accessing raw log resource attributes, drilling into a single host's metrics and processes, and tracing requests across your distributed microservices.
+
+To begin drilling down into a log, select the Date value that corresponds to your log of interest or use the advanced [Filter and Search]() feature to access your data.
+
+<Note> For more information on drilling into your logs, navigate to [Drilling Into Your Data](). </Note>
+
+# Log Properties
+
+The Log Properties section displays a componentized version of your raw JSON object: 
+
+![](https://lh7-us.googleusercontent.com/UEoX4SWceK7C5sfYo_BmTXKQjRV7bEOkEWOpo1NQPdFMKUkVOj0HaTJyg0SWkE_IYZWS7rID1aqBf7a4sOXcmIcMWuGKHR-3NPpGtIbk0tYgkWFOxIvTt0iVmr5QB9iHif9Z69If6ZPEYjJIj2cSCJY)
+
+# Raw
+
+The Raw section displays the JSON object for the specific log entry you are drilling into:
+
+![](https://lh7-us.googleusercontent.com/BL9KXXTTg-ckQ9aV1v_GxzInc6OTr8T62xS9Fp0Px4hCanVPth3fH7EGTx9pNZGxzWFLrWnPdktKNypYSRiINyctC4WEZRS0CmVm4Nyrj8pzAnNUxoPjDvobguPajmKXroCGjIRUNI_A4IW1Y-ViCec)
+
+# Metrics
+
+Within the Metrics section, there are three separate tabs relating to __Host__, __Process__, and __Container__ data.
+
+### Host
+
+The Host section displays all of your host data about the log you have drilled into. This tab is useful for identifying usage, network connection, and memory usage by state, and disk input/output on a per-host basis.
```

Comment:
For all three of these sections rather than saying "about the log you have drilled into" I suggest saying "about the host the log came from", because the log itself is not tied to these metrics explicitly